db/seeds.rb

```diff
@@ -84,2557 +84,26 @@ def require_updates?(real_attrs, seed_attrs)
-  description: "The first step to understanding design is to train yourself to *see* design, so you'll get a chance to break down one of the hallmark Design publications, Smashing Magazine.",
-  position: lesson_counter,
-  section_id: section.id,
-  is_project: true,
-  url: "/html_css/project_design.md",
-  repo: 'curriculum'
-)
-
-# +++++++++++
-# SECTION
-# +++++++++++
-
-section_position += 1
-section = create_or_update_section(
-  title: "Responsive Design and CSS Frameworks",
-  title_url: "Responsive Design and CSS Frameworks".parameterize,
-  course_id: course.id,
-  position: section_position,
-  description: "These days you need to make sure your pages display easily on multiple viewport sizes by using fluid layouts and media queries.  Luckily there are CSS frameworks like Twitter Bootstrap that can save you a ton of time developing standard pages and which come with responsive functionality for free."
-)
-
-lesson_counter += 1
-create_or_update_lesson(
-  title: "Responsive Design",
-  title_url: "Responsive Design".parameterize,
-  description: "Your websites will need to degrade gracefully as your users move from a full browser to an iPad to a mobile phone, and here you'll learn how.",
-  position: lesson_counter,
-  section_id: section.id,
-  is_project: false,
-  url: "/html_css/responsive_design.md",
-  repo: 'curriculum'
-)
-
-lesson_counter += 1
-create_or_update_lesson(
-  title: "Building with Responsive Design",
-  title_url: "Building with Responsive Design".parameterize,
-  description: "It's time to put your newfound responsive superpowers to use by building The Next Web's responsive homepage.",
-  position: lesson_counter,
-  section_id: section.id,
-  is_project: true,
-  url: "/html_css/project_responsive.md",
-  repo: 'curriculum'
-)
-
-lesson_counter += 1
-create_or_update_lesson(
-  title: "CSS Frameworks like Bootstrap and Foundation",
-  title_url: "CSS Frameworks like Bootstrap and Foundation".parameterize,
-  description: "Now that you've mastered the fundamentals of HTML and CSS, it's time to make your workflow a whole lot easier with CSS frameworks.",
-  position: lesson_counter,
-  section_id: section.id,
-  is_project: false,
-  url: "/html_css/css_frameworks.md",
-  repo: 'curriculum'
-)
-
-lesson_counter += 1
-create_or_update_lesson(
-  title: "Using Bootstrap",
-  title_url: "Using Bootstrap".parameterize,
-  description: "Test out working with the Bootstrap framework.  It may feel a bit odd at first but it makes your life MUCH easier once you figure out the gist of it. ",
-  position: lesson_counter,
-  section_id: section.id,
-  is_project: true,
-  url: "/html_css/project_bootstrap.md",
-  repo: 'curriculum'
-)
-
-# +++++++++++
-# SECTION
-# +++++++++++
-
-section_position += 1
-section = create_or_update_section(
-  title: "Advanced CSS3",
-  title_url: "Advanced CSS3".parameterize,
-  course_id: course.id,
-  position: section_position,
-  description: "We'll take you beyond the ba
```

Comment:
if we hard code the position of the course and section in each file and localize the lesson incriminators we can change this use the `count` method on the models instead:

```
Course.count
Section.count
Lesson.count
```